spec/controllers/external_authors_controller_spec.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe ExternalAuthorsController do
+  include LoginMacros
+  include RedirectExpectationHelper
+  let(:user) { FactoryGirl.create(:user) }
+  let(:invitation) { FactoryGirl.create(:invitation, external_author: external_author) }
+  let(:external_author) { FactoryGirl.create(:external_author) }
+
+  before(:each) do
```

Comment:
This works, but I think it also would have been okay to stick with repeating `fake_login_known_user(user)` where you specifically wanted to be logged in. DRY is good, but it forces you to add things like `fake_logout` and the special `before` and `after` blocks for the archivist examples, it might be _too_ DRY.